stable/external-dns/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Configure external DNS servers (AWS Route53, Google CloudDNS and others) for Kubernetes Ingresses and Services
+name: external-dns
+version: 0.0.1
+appVersion: v0.4.0-alpha.2
```

Comment:
Release 0.4.1 is out now: https://github.com/kubernetes-incubator/external-dns/releases